cmds/contest/server/server.go

```diff
@@ -173,7 +244,7 @@ func ServerMain(cmd string, args []string, sigs <-chan os.Signal) error {
 	plugins.Init(pluginRegistry, ctx.Logger())
```

Comment:
I think this initialises a hardcoded list of plugins again :/

Reply:
good point, will look into it

Reply:
Why do you think it's a hardcoded list? `pluginRegistry` is initialized with `pluginregistry.NewPluginRegistry` (which has an empty list of plugins), and only later populated in `server.Main` from the plugins written in the generated code

Reply:
Maybe I am reading it wrong, but won't `plugins.Init` end up here: https://github.com/facebookincubator/contest/blob/884d78e8b2a2f852997226df0ad0e14573f0d549/cmds/plugins/plugins.go#L66

And then load that hardcoded list of plugins?
Btw that thing also handles user functions for templates, they are basically plugins there.

Reply:
ooh you're right. If I'm reading that correctly, we don't need to call `Init` at all, because we call `registerPlugins` with the plugins initialized in `main.go`. So just removing it should be enough?

Reply:
I have removed it, and it seems it's only loading the plugins I expect. It also seems that the `cmds/plugins` package is not used anywhere else? Maybe it could be removed altogether?

Reply:
except in `db/rdbms/migration/0002_migrate_descriptor_to_extended_descriptor.go` but if I understand correctly it's OK to remove that file, and who needs migrations will check out earlier commits?